tests/test_blackd.py

```diff
@@ -8,12 +9,18 @@
 
 try:
     import blackd
-    from aiohttp.test_utils import AioHTTPTestCase, unittest_run_loop
+    from aiohttp.test_utils import AioHTTPTestCase
     from aiohttp import web
+except ImportError as e:
+    raise RuntimeError("Please install Black with the 'd' extra") from e
```

Comment:
The old `has_blackd_deps` logic was doing absolutely nothing useful so instead let's error out with an informative exception.